collections/_posts/2022-11-26-SparseMultiChannelVAE.md

```diff
@@ -0,0 +1,311 @@
+---
+layout: review
+title: "Sparse Multi-Channel Variational Autoencoder for the Joint Analysis of Heterogeneous Data"
+tags: VAE, Sparsity, Heterogeneous, Data
+author: "Olivier Bernard"
+cite:
+    authors: "Luigi Antelmi, Nicholas Ayache, Philippe Robert, Marco Lorenzi"
+    title:   "Sparse Multi-Channel Variational Autoencoder for the Joint Analysis of Heterogeneous Data"
+    venue:   "International Conference on Machine Learning (ICML) 2019"
+pdf: "http://proceedings.mlr.press/v97/antelmi19a/antelmi19a.pdf"
+---
+
+# Notes
+
+* Here are some (highly) useful links: [repo](https://github.com/ggbioing/mcvae), [video - slider on 1h05](https://youtube.videoken.com/embed/n5e2qNQ-h6E?tocitem=67), [slides](https://icml.cc/media/Slides/icml/2019/hallb(12-16-00)-12-17-00-5118-sparse_multi-ch.pdf)
+* This post has been jointly created by Olivier Bernard, Romain Deleat-Besson and Nathan Painchaud.
+
+&nbsp;
+
+# Highlights
+
+* The objective of this paper is to develop a formalism called sMCVAE (sparse multi-channel VAE) to handle with heterogenous data through VAE formulation.
+* This is achieved through two major innovations: 
+
+1) Variational distributions of each channel (type of data) are constrained to a common target prior in the latent space to bring interpretability.
+> this can be seen as a process of alignment 
+
+2) Parsimonious latent representations are enforced by ***variational dropout*** to make the method computationally advantageous and more easily interpretable.
+
+&nbsp;
+
+# Method
+
+## Starting point
+
+* Observations (e.g. patient information) are composed by subsets of information called channels as follows:
+
+$$\textbf{x}=\{\textbf{x}_1,\cdots,\textbf{x}_C\}$$ 
+
+$$\quad \quad$$ where each $$\textbf{x}_c$$ is a $$d_c$$-dimensional vector.
```

Comment:
Instead of manually adding spaces, you can simply indent the lines in markdown so that they'll be rendered as part of the current bullet item:
```suggestion
    $$\textbf{x}=\{\textbf{x}_1,\cdots,\textbf{x}_C\}$$ 

    where each $$\textbf{x}_c$$ is a $$d_c$$-dimensional vector.
```